Source/ImPlotWidgets/Public/ImPlotWidget.h

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+#pragma once
+
+#include <Runtime/Core/Public/CoreMinimal.h>
+#include <ImGui/Public/Widgets/ImGuiCustomWidget.h>
+
+struct FDataPoint
+{
+	float X = 0.0f;
+	float Y = 0.0f;
+};
+
+class IMPLOTWIDGETS_API SImPlotWidget : public SCompoundWidget
+{
+public:
+	SLATE_BEGIN_ARGS(SImPlotWidget)
+	{}
+	SLATE_ARGUMENT(TArray<TArray<FDataPoint>>, DataArrays)
+	SLATE_ARGUMENT(TArray<float>, InData)
```

Comment:
Is there a different way that we can do this, rather than having to add a new type of Data structure every time we make a new graph type? Did you look into potentially passing this in via the Construct function?

Reply:
We're going to add this in to a future change